CHANGELOG.md

```diff
@@ -15,9 +15,23 @@ Versioning](http://semver.org/spec/v2.0.0.html) except to the first release.
 - Support CRUD API (#108)
 - An ability to replace a base network connection to a Tarantool
   instance (#265)
+- Enumeration types for RLimitAction/iterators (#158)
```

Comment:
I'm not sure how GitHub works in this case, but it is possible that tickets would be closed after we merge these changes into the temporary branch. Is this the expected behavior?

Reply:
We will merge the branch into `master`. I will change the target branch and delete `v2_WIP` after `1.11.0` release.